__tests__/app.test.js

```diff
@@ -40,25 +41,141 @@ describe('GET /api', () => {
             .get('/api/mango')
             .expect(404)
             .then((response) => {
-                console.log(response.body)
-                expect(response.body.msg).toBe('404: Not Found')
+                expect(response.body.msg).toBe('Not Found')
             })
         })
     })
 
-    // describe('/api/articles/:article_id', () => {
-    //     test('200: Returns the desired article when given an article_id ', () => {
+    describe('GET /api/articles/:article_id', () => {
+        test('200: Returns the desired article when given an article_id ', () => {
+            
+            return request(app)
+            .get('/api/articles/3')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.article).toEqual(
+                {
+                    article_id: 3,
+                    title: "Eight pug gifs that remind me of mitch",
+                    topic: "mitch",
+                    author: "icellusedkars",
+                    body: "some gifs",
+                    created_at: '2020-11-03T09:12:00.000Z',
+                    votes: 0,
+                })
+
+            })
+            
+        });
+        test('400: Given an invalid data-type, send the client a 400 bad request.', () => {
+            return request(app)
+            .get('/api/articles/silly')
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toEqual('Bad Request')
+            })
+        })
+        test('404: Valid but non-existent ID', () => {
+            return request(app).get('/api/articles/999999')
+            .expect(404)
+            .then((response) => {
+                expect(response.body.msg).toEqual('Not Found')
+            })
+        })
+    });
+
+    describe('PATCH /api/articles/:article_id', () => {
+        test('200: Should update the relevant property by the given value of the specified id.', () => {
+
+            const updateVote = { inc_votes: 40 } 
+
+            return request(app)
+            .patch('/api/articles/3')
+            .expect(200)
+            .send(updateVote)
+            .then((response) => {
+                expect(response.body.update).toEqual(
+                    {
```

Comment:
I'm being picky but I probably wouldn't send back the updated article on a key of update. `updatedArticle` or even `article` would be great